llvm/include/llvm/ExecutionEngine/JITLink/x86_64.h

```diff
@@ -473,6 +486,15 @@ inline Error applyFixup(LinkGraph &G, Block &B, const Edge &E,
     break;
   }
 
+  case Delta8: {
+    int64_t Value = E.getTarget().getAddress() - FixupAddress + E.getAddend();
+    if (LLVM_LIKELY(isInt<8>(Value)))
+      *(little32_t *)FixupPtr = Value;
```

Comment:
I think `little32_t` will lead to overflow here. You can just write to `FixupPtr` without casting (it's already a `char*`), or make it explicit:
```c++
*(char*)FixupPtr = Value;
```